src/java/boa/functions/BoaAstIntrinsics.java

```diff
@@ -209,9 +210,12 @@ private static void openMap() {
 		final Configuration conf = new Configuration();
 		try {
 			final FileSystem fs = FileSystem.get(conf);
-			final Path p = new Path("hdfs://boa-njt/",
```

Comment:
I was actually running into this hard coded path problem with another branch.  I think we should just make this path configurable period - not only for local runners.  I was going to push a fix to master (very soon) to do just that which would make these changes obsolete.